From: Sergio Martins <iamsergio@gmail.com>
Date: Sun, 25 Nov 2012 15:45:22 +0000
Subject: Fix build with boost >= 1.48
X-Git-Url: http://quickgit.kde.org/?p=kdepim.git&a=commitdiff&h=e463ffacbaa4d722fe3b62039f2ad3560659f99a
---
Fix build with boost >= 1.48

For Qt 4.8.4 we might have a fix upstream.

QTBUG-22829
BUG: 304111
---


--- a/kleopatra/models/keylistmodel.cpp
+++ b/kleopatra/models/keylistmodel.cpp
@@ -61,7 +61,10 @@
 #define BOOST_ALL_NO_LIB
 #endif
 #include <boost/graph/adjacency_list.hpp>
+
+#ifndef Q_MOC_RUN // QTBUG-22829
 #include <boost/graph/topological_sort.hpp>
+#endif
 
 #include <algorithm>
 #include <vector>

